test/cp.js

```diff
@@ -757,7 +757,7 @@ test('should not overwrite recently created files (not give error no-force mode)
   t.is(shell.cat(`${t.context.tmp}/file1`).toString(), 'test1');
 });
 
-test('should copy fifos contents', t => {
+test('should copy fifo contents', t => {
```

Comment:
I think this case should also produce a warning. We'll completely drop fifo support and add it back in the plugin.